src/pages/Earn/useFarmRegistry.ts

```diff
@@ -82,21 +88,46 @@ export const useFarmRegistry = () => {
           rewardsUSDPerYear: farmData[e.returnValues.stakingAddress].rewardsUSDPerYear,
         })
       })
+
+    farmInfoEvents
```

Comment:
Instead of duplicating code, can we use `partition` to partition all the farms into 

const [featured, normal] = partition(...)